### PR TITLE
feat: add type definition

### LINKED ISSUE #6: typescript definition
Hi,

Thanks for the library, any plan to support/add type definition for this package?

### PR DESCRIPTION
**Pull request by bwgjoseph**
Added type definition for this project, closes #6 

I have tested with ts/js project, and both looking fine. As this my first time adding for type definition, I hope I didn't miss out anything.

If this does gets accepted, do consider adding `topic` and `label` for [hacktoberfest](https://hacktoberfest.digitalocean.com/hacktoberfest-update) so it counts towards the contribution

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   */
+  expireAfterSeconds?: number;
+  /**
+   * Define filename if want to persist data to file (uses feathers-nedb)
+   */
+  filename?: string;
+  /**
+   * Whether to expose UI on publicUrl (default is false) and debug without chrome extension
+   */
+  ui: boolean;
```
Comment (radenkovic) [inline types/index.d.ts]: this one is optional

File: types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  /**
+   * If UI is exposed, define custom url for debugger (default is /debugger)
+   */
+  publicUrl: string;
```
Comment (radenkovic) [inline types/index.d.ts]: this one is optional too

**Review by radenkovic — CHANGES_REQUESTED** [review]
This looks great so far, only two minor changes and we are good to go! Thanks for the PR!

**Comment by bwgjoseph** [comment]
Ah, yes. Thanks for that.

Changes are all done

**Comment by radenkovic** [comment]
Perfect, merging, thank you!

**Pull request merged**